src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -24,25 +24,6 @@ static constexpr bool _IsMouseMessage(UINT uMsg)
            uMsg == WM_MOUSEMOVE || uMsg == WM_MOUSEWHEEL || uMsg == WM_MOUSEHWHEEL;
 }
 
-// Helper static function to ensure that all ambiguous-width glyphs are reported as narrow.
-// See microsoft/terminal#2066 for more info.
-static bool _IsGlyphWideForceNarrowFallback(const std::wstring_view /* glyph */) noexcept
-{
-    return false; // glyph is not wide.
-}
-
-static bool _EnsureStaticInitialization()
-{
-    // use C++11 magic statics to make sure we only do this once.
-    static auto initialized = []() {
-        // *** THIS IS A SINGLETON ***
-        SetGlyphWidthFallback(_IsGlyphWideForceNarrowFallback);
-
-        return true;
-    }();
-    return initialized;
-}

```

Comment:
`CodepointWidthDetector` defaults to narrow glyphs if no fallback is specified already, so there's no need to do this.